.circleci/config.yml

```diff
@@ -144,7 +144,9 @@ jobs:
           name: build docker images and publish to Docker Hub
           environment:
             KO_DOCKER_REPO: honeycombio
-          command: ./build-docker.sh
+          command: |
+            echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
```

Comment:
```suggestion
            echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
```

Reply:
We removed the old env vars, these are the new ones to use in the Honeycomb Secrets context